### PR TITLE
Updated test data to dbSNP150 format (using chicken DB)

### PR DESCRIPTION
**Pull request by pabarcgar**

### CODE REVIEW

File: dbsnp-importer/src/test/resources/data.sql [hunk 1]
```diff
@@ -1,120 +1,38 @@
...
+INSERT INTO b150_contiginfo VALUES(966748672, 9031, 'NT_456093', 1, 'NT_456093.1', 'LGE64', 856351, 897575, 1, 966748672, 'Primary_Assembly', 'Gallus_gallus-5.0', NULL, 1, 964011153, 'KQ759333', 1, 103, 0, '2016-08-31 08:03:00.000', 1, 'GCF_000002315', 4, 966749097, NULL, 0, NULL, 41225);
+INSERT INTO b150_contiginfo VALUES(966748673, 9031, 'NT_456092', 1, 'NT_456092.1', 'LGE64', 429077, 856250, 0, 966748673, 'Primary_Assembly', 'Gallus_gallus-5.0', NULL, 1, 964011158, 'KQ759332', 1, 103, 0, '2016-08-31 08:03:00.000', 1, 'GCF_000002315', 4, 966749097, NULL, 0, NULL, 427174);
+INSERT INTO b150_contiginfo VALUES(966748945, 9031, 'NT_455837', 1, 'NT_455837.1', '3', 36431259, 58844808, 1, 966748945, 'Primary_Assembly', 'Gallus_gallus-4.0', NULL, 1, 964013905, 'KQ759055', 1, 103, 0, '2016-08-31 08:03:00.000', 1, 'GCF_000002315', 4, 966749129, NULL, 0, NULL, 22413550);
+
+
+-- columns snp_id, avg_heterozygosity, het_se, create_time, last_updated_time, cpg_code, tax_id, validation_status, exemplar_subsnp_id, univar_id, cnt_subsnp, map_property
```
Comment (cyenyxe) [inline dbsnp-importer/src/test/resources/data.sql]: Do we need to populate this table? It is not queried by the reader.

File: dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java [hunk 2]
```diff
@@ -139,20 +158,18 @@ public void testQuery() throws Exception {
...
+
+        assertEquals(1, list.size());
+        assertEquals(snpInDifferentAssembly, list.get(0));
     }
 
     @Test
     public void testQueryWithDifferentAssemblyType() throws Exception {
-        SubSnpCoreFieldsReader fieldsReader = buildReader("Btau_5.0.1", Collections.singletonList("non-nuclear"), 2000);
-        List<SubSnpCoreFields> list = readAll(fieldsReader);
+        reader = buildReader(CHICKEN_ASSEMBLY_5, Collections.singletonList("non-nuclear"), PAGE_SIZE);
```
Comment (cyenyxe) [inline dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java]: "non-nuclear" would also be a good constant, because it is one of the usual assembly types along with "Primary_assembly".

File: dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java [hunk 3]
```diff
@@ -139,20 +158,18 @@ public void testQuery() throws Exception {
...
-        assertEquals(2, list.size());
-        for (SubSnpCoreFields subSnpCoreFields : list) {
-            assertNotNull(subSnpCoreFields);
-            assertTrue(expectedRsIds.containsKey(subSnpCoreFields.getRsId()));
-        }
+        reader = buildReader(CHICKEN_ASSEMBLY_4, Collections.singletonList(PRIMARY_ASSEMBLY), PAGE_SIZE);
+        List<SubSnpCoreFields> list = readAll(reader);
+
+        assertEquals(1, list.size());
+        assertEquals(snpInDifferentAssembly, list.get(0));
```
Comment (cyenyxe) [inline dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java]: Since this is testing an "edge case", maybe the variant could be initialized in here instead of making it an attribute? That would also make the description in the `setUp` method unnecessary.

Comment (pabarcgar) [reply dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java]: Done in 		d84bca9
